docs/documentation/stories/application-environments.md

```diff
@@ -1,48 +1,73 @@
 # Application Environments
 
-## Configuring available environments
+In Angular CLI you can configure the build system to replace existing files for your intended
+environment.
 
-`.angular-cli.json` contains an **environments** section.  By default, this looks like:
+## Configuring available file replacements
 
-``` json
-"environments": {
-    "dev": "environments/environment.ts",
-    "prod": "environments/environment.prod.ts"
-}
-```
-
-You can add additional environments as required.  To add a **staging** environment, your configuration would look like:
+`angular.json` contains an **fileReplacements** section within the production configuration of the
+build target. By default, this looks like:
 
 ``` json
-"environments": {
-    "dev": "environments/environment.ts",
-    "staging": "environments/environment.staging.ts",
-    "prod": "environments/environment.prod.ts"
+"configurations": {
+  "production": {
+    "fileReplacements": [
+      {
+        "replace": "src/environments/environment.ts",
+        "with": "src/environments/environment.prod.ts"
+      }
+    ],
+```
```

Comment:
Add closing curly braces for readability.

Reply:
Added `...` to indicate there are more options in the prod config.